sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -224,7 +224,7 @@ The following section provides several code snippets covering some of the most c
 - [Recognize PII Entities](#recognize-pii-entities "Recognize pii entities")
 - [Extract Key Phrases](#extract-key-phrases "Extract key phrases")
 - [Detect Language](#detect-language "Detect language")
-- [Healthcare Analysis](#healthcare-analysis "Healthcare analysis")
+- [Healthcare Entities Analysis](#healthcare-analysis "Healthcare analysis")
```

Comment:
I think this needs to be `[Healthcare Entities Analysis](#healthcare-entities-analysis "Healthcare Entities Analysis")` or the link won't work

Reply:
Fixed!